src/wallet.js

```diff
@@ -876,6 +876,51 @@ const wallet = {
     return data;
   },
 
+  /**
+   * Filter an utxo based on the specified options
```

Comment:
Add a note of what functions are using it.

Reply:
Done